src/language/pattern-types.md

```diff
@@ -7,6 +7,97 @@ This page is a reference for the different kinds of patterns.
 For an overview of how patterns work, where you can use them in Dart, and common
 use cases, visit the main [Patterns][] page.
 
+#### Pattern precedence
+
+Similar to [operator precedence](/language/operators#operator-precedence-example),
+pattern evaluation adheres to precendence rules.
+You can use [parenthesized patterns](#parenthesized) to evaluate lower-precedence
+patterns first.  
+
+We've listed the patterns ascending order of precedence:
```

Comment:
```suggestion
We've listed the patterns ascending order of precedence:

```

Reply:
Actually, I'm reading this again and I feel some word is missing?

```suggestion
This document lists the pattern types in ascending order of precedence.

```

Reply:
Oh yeah, it should have been "..._in_ ascending...". I like "This document..." much better than "We've...", so thanks !